addOns/pscanrulesAlpha/src/test/java/org/zaproxy/zap/extension/pscanrulesAlpha/SourceCodeDisclosureScannerUnitTest.java

```diff
@@ -19,27 +19,106 @@
  */
 package org.zaproxy.zap.extension.pscanrulesAlpha;
 
-import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.is;
 import static org.junit.Assert.assertThat;
 
+import java.io.IOException;
+import org.junit.Before;
 import org.junit.Test;
+import org.parosproxy.paros.Constant;
+import org.parosproxy.paros.core.scanner.Alert;
+import org.parosproxy.paros.network.HttpMessage;
 
 public class SourceCodeDisclosureScannerUnitTest
         extends PassiveScannerTest<SourceCodeDisclosureScanner> {
 
+    private static final String CODE_SQL = "insert into vulnerabilities values(";
+    private static final String CODE_PHP = "<?php echo 'evils'; ?>";
+    private static final String CODE_HTML = "<p>Innocent HTML</p>";
+    private static final String URI = "https://www.example.com";
+
+    private HttpMessage msg;
+
     @Override
     protected SourceCodeDisclosureScanner createScanner() {
         return new SourceCodeDisclosureScanner();
     }
 
     @Test
-    public void scannerNameShouldMatch() {
-        // Quick test to verify scanner name which is used in the policy dialog but not
-        // alone in alerts
+    public void givenInnocentHtmlBodyThenNoAlertRaised() {
```

Comment:
`givenJustHtmlThen...` or `givenNoServerSideCodeThen...`, not sure the "Innocent" remark is necessary.